packages/injective-cosmwasm/src/exchange_mock_querier.rs

```diff
@@ -38,8 +38,9 @@ pub fn mock_dependencies() -> OwnedDeps<MockStorage, MockApi, WasmMockQuerier, I
 fn default_subaccount_deposit_response_handler() -> QuerierResult {
     let response = SubaccountDepositResponse {
         deposits: Deposit {
-            available_balance: FPDecimal::from(100u128),
-            total_balance: FPDecimal::from(100u128),
+            // NOTE: this is 100 with 8 decimal places
```

Comment:
maybe it would be better to return different value based whether it's quote or base asset? because for quote this will still be a very low value

Reply:
I mean decimal wise... for INJ 100 with 8 zeros is still a very low amount